api/proto/certs/certs.proto

```diff
@@ -43,6 +43,6 @@ enum ZCertType {
 }
 
 message ZCertProperties {
-   bool isMutable = 1; //if this is set, this certificate can't be overwritten
+   bool isMutable = 1; //set to false for immutable certificates
```

Comment:
What's the rationale for this knob?

Reply:
This is to prevent an attacker on the EVE node from replacing current certificate by a new certificate(which may not be based on TPM, let's say), and use that certificate to send attestation quote of his choice, without involving TPM at all.  For this reason, we need a notion where we can say, this certificate is added once, and can never be replaced with another certificate of the same type.

Reply:
Got it. Can you please add this explanation to the documentation @cshari-zededa ?

Reply:
Updated API.md